### DISCUSSION TITLE
Automatically removing specified packages

### DISCUSSION THREAD

**Discussion by sodle-splunk** [renovatebot/renovate#28171] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Hi! Can Renovate automatically _remove_ a specified set of packages? 

For example, we have a bunch of Python repos that used to depend on `requests` but don't anymore. Can I create a Renovate rule to remove it from `requirements.txt`?

**Comment by rarkins** [comment, marked as answer]
No, Renovate doesn't have that capability
